### ISSUE TITLE
Please update stable to 1.36.0

### ISSUE THREAD

**Issue by cmdln** [emk/rust-musl-builder#76]
**How could this project be improved?**
1.36.0 was just released. Would be great to have an image for this release.

Also, it looks like sqlite in the 1.35.0 is not up to date. I get this error when trying to build diesel with the sqlite feature:
```
  = note: /usr/bin/ld: cannot find -lsqlite3
          collect2: error: ld returned 1 exit status
```

I can get around this by run `apt update && apt install sqlite3 libsqlite3-dev libzip-dev`. I added the dev package for libzip because after adding sqlite, I ran into a similar error but with -`lz`.

**If you're interested in implementing this feature yourself, how could I help you?**
Happy to attempt a PR, if that helps.

**Comment by cmdln** [comment]
I updated with a little more detail with some issues I had with the existing 1.35.0 image.

**Comment by cmdln** [comment]
I am having another issue, I am waiting on `./test-image.sh` to complete before submitting that bug as I am not 100% certain updating stable will address what I am seeing.

**Comment by emk** [comment]
This should have been updated a little while ago. I hope everything is working!
